### PR TITLE
Update Github Actions

### PR DESCRIPTION
**Pull request by denisandreenko**

### CODE REVIEW

File: .github/settings.yml [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
+# SPDX-License-Identifier: Apache-2.0
+#
+
 repository:
   name: firefly-tezosconnect
+  description: Firefly Tezos Connect
+  homepage: https://github.com/hyperledger/firefly-tezosconnect
   default_branch: main
-  has_downloads: false
+  has_downloads: true
```
Comment (nguyer) [inline .github/settings.yml:10]: What downloads will this repo have?

File: .github/settings.yml [hunk 2]
```diff
@@ -1,12 +1,17 @@
...
   has_issues: true
   has_projects: true
-  has_wiki: false
+  has_wiki: true
```
Comment (nguyer) [inline .github/settings.yml:13]: Do we have content for a Tezosconnect specifici wiki? Typically we've been steering away from repo-specific Wikis and putting everything in the main FireFly docs site.

**Review by nguyer — APPROVED** [review]
Just a couple of questions on the repo settings, but otherwise looks fine. If there are good reasons for the repo settings, feel free to merge as-is.

File: .github/settings.yml (hunk 2 above)
Comment (denisandreenko) [reply .github/settings.yml:13]: I've found this option for EVM Connect and thought that it's about the main FireFly docs site.
But now I realised that it's about specific project wiki.
Thanks for that. (The option needs to be updated for EVM Connector as well)

File: .github/settings.yml (hunk 1 above)
Comment (denisandreenko) [reply .github/settings.yml:10]: I assume this option is about releases tar files,
which will appear soon when the service release.

**Pull request merged**